src/measure_theory/measurable_space.lean

```diff
@@ -1283,11 +1290,81 @@ begin
   exact (measurable_embedding.subtype_coe hf₂).comp e.measurable_embedding
 end
 
-lemma of_measurable_inverse {g : β → α} (hf₁ : measurable f)
+lemma of_measurable_inverse (hf₁ : measurable f)
   (hf₂ : measurable_set (range f)) (hg : measurable g)
   (H : left_inverse g f) : measurable_embedding f :=
 of_measurable_inverse_on_range hf₁ hf₂ (hg.comp measurable_subtype_coe) H
 
+open_locale classical
+
+/-- The *`measurable Schröder-Bernstein Theorem*: Given measurable embeddings
+`α → β` and `β → α`, we can find a measurable equivalence `α ≃ᵐ β`.-/
+noncomputable
+def schroeder_bernstein {f : α → β} {g : β → α}
+  (hf : measurable_embedding f)(hg : measurable_embedding g) : (α ≃ᵐ β) :=
+begin
+  let F : set α → set α := λ A, (g '' (f '' A)ᶜ)ᶜ,
+  --We follow the proof of the usual SB theorem in mathlib,
+  --the crux of which is finding a fixed point of this F.
+  --However, we must find this fixed point manually instead of invoking Knaster-Tarski
+  --in order to make sure it is measurable.
+  suffices : Σ' A : set α, measurable_set A ∧ F A = A,
+  { rcases this with ⟨A, Ameas, Afp⟩,
+    let B := f '' A,
+    have Bmeas : measurable_set B := hf.measurable_set_image' Ameas,
+    refine (measurable_equiv.sum_compl Ameas).symm.trans
+      (measurable_equiv.trans _ (measurable_equiv.sum_compl Bmeas)),
+    apply measurable_equiv.sum_congr,
+    { exact hf.equiv_image _, },
+    have : Aᶜ = g '' Bᶜ,
+    { apply compl_injective,
+      rw ← Afp,
+      simp, },
+    rw this,
+    exact (hg.equiv_image _).symm, },
+  have Fmono : ∀ {A B}, A ⊆ B → F A ⊆ F B,
+  { intros A B hAB,
+    rw compl_subset_compl,
+    apply set.image_subset,
+    rw compl_subset_compl,
+    apply set.image_subset,
+    exact hAB, },
+  let X : ℕ → set α :=
+  begin
+    intro n,
+    induction n with n ih,
+    { exact univ },
+    exact F ih,
+  end,
+  use Inter X, split,
```

Comment:
```suggestion
  use Inter X,
  split,
```
Only one tactic per line. An alternative for those two lines is `refine ⟨Inter X, _, _⟩,`